Templates/TypeScript/src/entity_types.ts.tt

```diff
@@ -9,45 +10,159 @@
     var entityTypes             = model.GetEntityTypes();
     var enumTypes               = model.GetEnumTypes();
     var complexTypes            = model.GetComplexTypes();
+
+    var maxLineLength = 180;
```

Comment:
180 is high, let's make it 120.

Reply:
The allowed max length by definitely typed repo is 200, I just shortened it 180, in case tabs adds up some length to the line

Reply:
Ideally, we don't want comments over 80 chars in length since it requires more eye movement across the screen. 120 should be the max.

Reply:
Okay will fix this.